satellite/contact/service.go

```diff
@@ -17,24 +22,78 @@ var Error = errs.Class("contact")
 
 var mon = monkit.Package()
 
+// Config contains configurable values for contact service
+type Config struct {
+	ExternalAddress string `user:"true" help:"the public address of the node, useful for nodes behind NAT" default:""`
+}
+
+// Conn represents a connection
+type Conn struct {
+	conn   *grpc.ClientConn
+	client pb.NodesClient
+}
+
 // Service is the contact service between storage nodes and satellites.
 // It is responsible for updating general node information like address, capacity, and uptime.
 // It is also responsible for updating peer identity information for verifying signatures from that node.
 //
 // architecture: Service
```

Comment:
whats up with this `architecture: Service` comment? is this a new process to add this?

Reply:
same as above